### PR TITLE
Use mocks for TestInstanceStats_Get and TestObjectStorage_Cancel tests

### PR DESCRIPTION
**Pull request by amisiorek-akamai**
## 📝 Description

Makes mocks for TestInstanceStats_Get and TestObjectStorage_Cancel such that we no longer have to skip those tests.

## ✔️ How to Test

`make test SKIP_LINT=1 ARGS="-run TestInstanceStats_Get"`

`make test SKIP_LINT=1 ARGS="-run TestObjectStorage_Cancel"`

### CODE REVIEW

**Comment by zliang-akamai** [comment]
Can we create a unit test directory to hold all mocked tests or maybe making a `xxxx_test.go` file in the root directory of the repo (like [this one](https://github.com/linode/linodego/blob/main/client_test.go))?

**Comment by zliang-akamai** [comment]
@amisiorek-akamai Ah yes, your are right, didn't realize that, lol

**Review by zliang-akamai — APPROVED** [review]
Tested locally and passed!

File: test/integration/instance_stats_test.go [hunk 1]
```diff
@@ -2,52 +2,49 @@ package integration
...
+				PrivateIn: [][]float64{},
+				PrivateOut: [][]float64{},
+			},
+			NetV6: linodego.StatsNet{
+				In: [][]float64{},
+				Out: [][]float64{},
+				PrivateIn: [][]float64{},
+				PrivateOut: [][]float64{},
+			},
+		},
```
Comment (lgarber-akamai) [inline test/integration/instance_stats_test.go:36]: Could this be populated with some arbitrary data just to make sure everything unmarshals correctly?

**Review by lgarber-akamai — CHANGES_REQUESTED** [review]

**Review by lgarber-akamai — APPROVED** [review]
Looks good and tests pass locally, great work!

**Pull request merged**